server/MoqGenerator/Services/InterfaceGenericsBuilder.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using MoqGenerator.Interfaces.Lsp;
+using MoqGenerator.Model;
+
+namespace MoqGenerator.Services
+{
+	public class InterfaceGenericsBuilder : IInterfaceGenericsBuilder
```

Comment:
this isn't registered in C.\speedy-moq\server\OmniLsp\Program.cs ?? was that just a miss?